bzt/modules/console.py

```diff
@@ -1165,12 +1175,25 @@ def update(self):
                 if eta >= 0:
                     self.eta.set_text("ETA: %s" % humanize_time(eta))
                 else:
-                    over = elapsed - self.duration
-                    self.eta.set_text("Overtime: %s" % humanize_time(over))
-            else:
-                self.eta.set_text("")
+                    self.eta.set_text("Overtime: %s" % humanize_time(-eta))
 
-            if isinstance(self.progress, ProgressBar):
-                # noinspection PyUnresolvedReferences
                 self.progress.set_completion(elapsed)
+            else:
+                self.progress.set_text("Running...")
+
+            if self.executor.check():
```

Comment:
You cannot call executor.check(), it's meant to be called only from core/provisioning.